packages/hint/src/lib/enums/category.ts

```diff
@@ -1 +0,0 @@
-export * from '@hint/utils/dist/src/types/category';
```

Comment:
Is this file deleted? If not, shouldn't it?

Reply:
It is deleted.